pgex/examples/glow_effect/main.py

```diff
@@ -0,0 +1,93 @@
+"""
```

Comment:
There should be an `__init__.py` file (so that the example directory is a package) that runs the file on import (see other examples)